TeamCode/src/main/java/org/firstinspires/ftc/teamcode/core/lib/autonomousControl/MotorVelocityData.java

```diff
@@ -0,0 +1,43 @@
+package org.firstinspires.ftc.teamcode.core.lib.autonomousControl;
+
+import org.firstinspires.ftc.teamcode.robot.constants.AutonomousConstants;
+
+public class MotorVelocityData {
+    public double velocityFrontLeft;
+    public double velocityFrontRight;
+    public double velocityBackLeft;
+    public double velocityBackRight;
+    public MotorVelocityData(){
+
+    }
+    public MotorVelocityData(double vFL,double vFR, double vBL,double vBR){
+        velocityFrontLeft = vFL;
+        velocityFrontRight = vFR;
+        velocityBackLeft = vBL;
+        velocityBackRight = vBR;
+    }
+    public MotorVelocityData updateAppliedVelocities (Pose2d botRelativeVelocitiy){
+        double botVX = botRelativeVelocitiy.getX();
+        double botVY = botRelativeVelocitiy.getY();
+        double Vheading = botRelativeVelocitiy.getHeadingRadians();
+        if (Math.abs(Vheading) > AutonomousConstants.MAX_HEADING_VELOCITY){
```

Comment:
use ternary instead
fy update to vHeading

Reply:
I didn't actually know that ternary was a thing in java, I thought it was a Python thing. But nonetheless I implemented said changes among other on the latest commit